code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,773 @@
+	icon = 'icons/obj/card.dmi'
+	icon_state = "syndie"
+	access = list(ACCESS_AWAY01)
+
+/obj/item/card/id/away/dsbssigma/private
+	registered_name = "William Simmons"
+	assignment = "Private"
+	access = list(ACCESS_AWAY01, ACCESS_AWAY02)
+
+/obj/item/card/id/away/dsbssigma/officer
+	registered_name = "Xiu Yao"
+	assignment = "Officer"
+	access = list(ACCESS_AWAY01, ACCESS_AWAY02, ACCESS_AWAY03)
+
+/obj/item/organ/internal/heart/gland/heals/weak
+	name = "fleshy growth"
+	desc = "A nausea-inducing growth of flesh."
+	origin_tech = "biotech=5"
+
+/obj/item/organ/internal/heart/gland/heals/weak/activate()
+	to_chat(owner, "<span class='notice'>You feel slightly revitalized.</span>")
+	owner.adjustToxLoss(-10)
+	owner.adjustBruteLoss(-10)
+	owner.adjustOxyLoss(-10)
+	owner.adjustFireLoss(-10)
+
+/obj/item/ammo_casing/c10mm/spent
+	projectile_type = null
+
+/obj/item/ammo_casing/c45/spent
+	projectile_type = null
+
+////// MACHINARY //////
+
+/obj/machinery/computer/id_upgrader/dsbssigma
+	icon_keyboard = "laptop_key"
+	access_to_give = list(ACCESS_AWAY01)
+
+/obj/machinery/vending/medical/syndicate_access/dsbssigma
+	req_access = list(ACCESS_AWAY01)
+	products = list(/obj/item/reagent_containers/syringe = 4, /obj/item/reagent_containers/food/pill/patch/styptic = 2, /obj/item/reagent_containers/food/pill/patch/silver_sulf = 2, /obj/item/reagent_containers/applicator/brute = 1, /obj/item/reagent_containers/applicator/burn = 1,
+					/obj/item/reagent_containers/glass/bottle/charcoal = 1, /obj/item/reagent_containers/glass/bottle/salicylic = 1, /obj/item/reagent_containers/glass/bottle/saline = 1, /obj/item/reagent_containers/glass/bottle/ether = 1,
+					/obj/item/reagent_containers/glass/bottle/atropine = 1, /obj/item/reagent_containers/glass/bottle/oculine = 1, /obj/item/reagent_containers/glass/bottle/toxin = 1, /obj/item/reagent_containers/syringe/antiviral = 2,
+					/obj/item/reagent_containers/syringe/insulin = 1, /obj/item/reagent_containers/hypospray/autoinjector = 2, /obj/item/reagent_containers/food/pill/salbutamol = 2, /obj/item/reagent_containers/food/pill/mannitol = 2, /obj/item/reagent_containers/food/pill/mutadone = 1,
+					/obj/item/stack/medical/bruise_pack/advanced = 1, /obj/item/stack/medical/ointment/advanced = 1, /obj/item/stack/medical/bruise_pack = 2, /obj/item/stack/medical/splint = 2, /obj/item/reagent_containers/glass/beaker = 2, /obj/item/reagent_containers/dropper = 1,
+					/obj/item/healthanalyzer = 1, /obj/item/healthupgrade = 1, /obj/item/reagent_containers/iv_bag/slime = 1)
+	contraband = list(/obj/item/reagent_containers/glass/bottle/sulfonal = 1, /obj/item/reagent_containers/glass/bottle/pancuronium = 1)
+
+/obj/machinery/autolathe/hacked
+	hacked = 1
+
+/obj/machinery/light/broken
+	active_power_usage = 0
+	icon_state = "tube-broken"
+	status = 2
+
+/obj/machinery/light/small/broken
+	active_power_usage = 0
+	icon_state = "bulb-broken"
+	status = 2
```

Comment:
These should be put in the same file as the normal ones I feel. Since these are reusable in other places as well